debian/rules

```diff
@@ -22,6 +22,13 @@ include /usr/share/dpkg/default.mk
 %:
 	dh $@ --with python2
 
+# This target updates pycam/Version.py and debian/changelog with current
+# information from git.  It's intended to be run before generating the
+# debian source package.
+prep-source:
+	$(MAKE) pycam/Version.py
+	debian/update-dch-from-git
```

Comment:
Personally I would prefer to stick to Debian's packaging policy, even though this is an inofficial packaging (for now).
Thus automated manipulation of the Debian changelog within `debian/rules` feels inappropriate to me. I think this should happen in "our" Makefile instead.